prelude/bootstrap.js

```diff
@@ -2116,7 +2116,10 @@ function payloadFileSync(pointer) {
         // Example: /tmp/pkg/<hash>/sharp/build/Release/sharp.node
         newPath = path.join(tmpFolder, modulePackagePath, moduleBaseName);
       } else {
-        // simple load the file in the temporary folder
+        createDirRecursively(tmpFolder);
+        copyFileSync(modulePath, tmpFolder);
```

Comment:
Just out of curiosity: there used to be an `if` check to see if the file already existed, is this not necessary anymore?

Reply:
See #1492.